### PR TITLE
mention migration, cosmetics

### PR DESCRIPTION
**Pull request by fwolfst**
I was searching on what the order_column should look like (rtfm) and did not immediately (light speed) find it, so I guess putting the instructions in a more declarative manner (and with syntax highlight) might make sense. Besides, include a sample migration for existing data (#75).

The rest is cosmetics.

### CODE REVIEW

File: Readme.mkd [hunk 1]
```diff
@@ -1,13 +1,13 @@
-**ranked-model** is a modern row sorting library built for Rails 3, 4 and 5.  It uses ARel aggressively and is better optimized than most other libraries.
+**ranked-model** is a modern row sorting library built for Rails 3+.  It uses ARel aggressively and is better optimized than most other libraries.
```
Comment (brendon) [inline Readme.mkd]: It's actually for Rails 4.2+ :)

File: Readme.mkd [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 [![Build Status](https://travis-ci.org/mixonic/ranked-model.png)](https://travis-ci.org/mixonic/ranked-model)
 
 Installation
 ------------
 
-ranked-model passes specs with Rails 4.2, 5.0, 5.1, 5.2, 6.0 for MySQL, Postgres, and SQLite on Ruby 2.4 through 2.6, and jruby-9.1.17.0 where Rails supports the platform. This is with the exception of Postgres before Rails 4.0 on all platforms, which is unsupported by `ranked-model`. Note that the `pg` gem has pulled support for rbx (Rubinius) from version 1 onward.
+ranked-model passes specs with Rails 4.2, 5.0, 5.1, 5.2 and 6.0 for MySQL, Postgres, and SQLite on Ruby 2.4 through 2.6, and jruby-9.1.17.0 where Rails supports the platform. This is with the exception of Postgres before Rails 4.0 on all platforms, which is unsupported by `ranked-model`. Note that the `pg` gem has pulled support for rbx (Rubinius) from version 1 onward.
 
-TL;DR, if you are using Rails 4 and up you are 100% good to go. Before Rails 4, be wary of Postgres.
+TL;DR, **if you are using Rails 4 and up you are 100% good to go.** Before Rails 4, be wary of Postgres.
```
Comment (brendon) [inline Readme.mkd]: We can remove this line.

File: Readme.mkd [hunk 3]
```diff
@@ -1,13 +1,13 @@
-**ranked-model** is a modern row sorting library built for Rails 3, 4 and 5.  It uses ARel aggressively and is better optimized than most other libraries.
+**ranked-model** is a modern row sorting library built for Rails 3+.  It uses ARel aggressively and is better optimized than most other libraries.
 
 [![Build Status](https://travis-ci.org/mixonic/ranked-model.png)](https://travis-ci.org/mixonic/ranked-model)
 
 Installation
 ------------
 
-ranked-model passes specs with Rails 4.2, 5.0, 5.1, 5.2, 6.0 for MySQL, Postgres, and SQLite on Ruby 2.4 through 2.6, and jruby-9.1.17.0 where Rails supports the platform. This is with the exception of Postgres before Rails 4.0 on all platforms, which is unsupported by `ranked-model`. Note that the `pg` gem has pulled support for rbx (Rubinius) from version 1 onward.
+ranked-model passes specs with Rails 4.2, 5.0, 5.1, 5.2 and 6.0 for MySQL, Postgres, and SQLite on Ruby 2.4 through 2.6, and jruby-9.1.17.0 where Rails supports the platform. This is with the exception of Postgres before Rails 4.0 on all platforms, which is unsupported by `ranked-model`. Note that the `pg` gem has pulled support for rbx (Rubinius) from version 1 onward.
```
Comment (brendon) [inline Readme.mkd]: Can also remove the part about 'before Rails 4.0'.

**Comment by brendon** [comment]
Looking good :) Just a few tweaks :)

**Comment by fwolfst** [comment]
I will look into it in a calm moment; currently its crazy times. Like "No time(s)".

Btw, the description of the github repo also states "An acts_as_sortable/acts_as_list replacement built for Rails 3 & 4". :D

**Comment by brendon** [comment]
Sure is crazy times. The whole of NZ is just stuck at home for 4 weeks. Unfortunately I can't change the description until @mixonic transfers this project over to me. :) Were you still keen to do this @mixonic?

File: Readme.mkd [hunk 4]
```diff
@@ -172,6 +179,32 @@ truck.row_order
...
+```ruby
+# e.g. file db/migrate/20200325095038_add_row_order_to_ducks.rb
+class AddRowOrderToDucks < ActiveRecord::Migration[6.0]
+  def change
+    add_column :courses, :row_order, :integer
+
+    # Newest Duck shall rank "highest"" (be last).
+    Duck.order(created_at: :desc).desc.each do |duck|
+      duck.update!(row_order: duck.created_at.to_i)
+    end
```
Comment (tjwallace) [inline Readme.mkd]: Might be able to do this with one SQL statement:
`Duck.update_all('row_order = EXTRACT(EPOCH FROM created_at)')`

**Comment by fwolfst** [comment]
afaics, I implemented all proposed changes. Thanks @tjwallace . Although I'd keep the procedural example commented out.

**Comment by brendon** [comment]
Excellent @fwolfst, thanks for your hard work! :D
